### ISSUE TITLE
HTML conversion from Markdown

### ISSUE THREAD

**Issue by rgaiacs** [swcarpentry/DEPRECATED-lesson-template#59]
@gvwilson or @aaren Could you help me discovery why when I build our lessons some lines of the HTML change even if the Markdown files are the same.

```
$ git log HEAD -1
commit 142672a263f64212d2f7608bf1ac7f8db1031d77
Author: Greg Wilson <gvwilson@third-bit.com>
Date:   Fri Dec 12 20:40:42 2014 -0500

    Including compiled HTML pages
$ pandoc --version | head -n 1
pandoc 1.13.2
$ make -B preview
...
$ git diff
diff --git a/01-one.html b/01-one.html
index 7e3b0f7..bf85a39 100644
--- a/01-one.html
+++ b/01-one.html
@@ -28,7 +28,7 @@
           <h1 class="title">Lesson Title</h1>
           <h2 class="subtitle">Topic Title One</h2>
 <div id="learning-objectives" class="objectives">
-<h2>Learning Objectives</h2>
+<h2 id="learning-objectives" class="objectives">Learning Objectives</h2>
 <ul>
 <li>Learning objective 1</li>
 <li>Learning objective 2</li>
diff --git a/02-two.html b/02-two.html
index ea737d2..476bf48 100644
--- a/02-two.html
+++ b/02-two.html
@@ -28,7 +28,7 @@
           <h1 class="title">Lesson Title</h1>
           <h2 class="subtitle">Topic Title Two</h2>
 <div id="learning-objectives" class="objectives">
-<h2>Learning Objectives</h2>
+<h2 id="learning-objectives" class="objectives">Learning Objectives</h2>
 <ul>
 <li>Learning objective 1</li>
 <li>Learning objective 2</li>
diff --git a/index.html b/index.html
index 05466f4..7433b96 100644
--- a/index.html
+++ b/index.html
@@ -29,7 +29,7 @@

 <p>Paragraph of introductory material.</p>
 <div id="prerequisites" class="prereq">
-<h2>Prerequisites</h2>
+<h2 id="prerequisites" class="prereq">Prerequisites</h2>
 <p>A short paragraph describing what learners need to know before tackling this lesson.</p>
 </div>
 <h2 id="topics">Topics</h2>
```

**Comment by aaren** [comment]
I don't get this, but I have `pandoc 1.13.1`.

The difference you are getting is that your headers are having attributes placed on them.

I think I've got pandoc 1.13.2 around somewhere - where did you install this from?

**Comment by rgaiacs** [comment]
> I don't get this, but I have `pandoc 1.13.1`.

Thanks for this info.

> I think I've got pandoc 1.13.2 around somewhere - where did you install this from?

When I start playing with Pandoc for Software Carpentry I found some bugs that I
reported to Pandoc. When this bugs were fixed I install Pandoc from source to
check if the bugs were fixed and since that I install Pandoc from source.

At the end of the week I will have some time to debug this and send a pull
request for us or report this to Pandoc developers.

**Comment by aaren** [comment]
I can build pandoc 1.13.2, but without knowing exactly which commit you built it from it might be hard to find the issue.

Thinking more widely, I think we need to stipulate a minimum pandoc version to be used. This would be a bit easier if there were linux builds available as easily as windows.

**Comment by rgaiacs** [comment]
> I can build pandoc 1.13.2, but without knowing exactly which commit you built it from it might be hard to find the issue.

Before I open this issue I built pandoc with the last changes available at
https://github.com/jgm/pandoc, so the exactly commit is
https://github.com/jgm/pandoc/commit/4ffa70970dc7219aa71c137c728280b7460ed604.

Probably I'm getting this behavior due https://github.com/jgm/pandoc/issues/1711
(the last bug that I report to Pandoc).

> Thinking more widely, I think we need to stipulate a minimum pandoc version to be used.
> This would be a bit easier if there were linux builds available as easily as windows.

Most Linux distributions have Pandoc available
([Debian has 1.9.4.2](https://packages.debian.org/wheezy/pandoc)) and for
Windows you can always go to https://github.com/jgm/pandoc/releases.

**Comment by aaren** [comment]
Pandoc 1.13.2 has just been [released](https://github.com/jgm/pandoc/releases/tag/1.13.2). With this version of Pandoc I can reproduce what you're getting. Not sure why just yet - will look into it.

Note that the new release includes a .deb for 64bit Debian / Ubuntu, meaning it is easy to install an up to date pandoc on Windows, Mac and the majority of GNU/Linux systems now (no Haskell platform necessary!).

**Comment by aaren** [comment]
@r-gaia-cs I just fixed this in #93 but now notice that you've fixed in #85 as well.
